evalml/automl/automl_search.py

```diff
@@ -607,10 +607,18 @@ def _compute_cv_scores(self, pipeline, X, y):
         start = time.time()
         cv_data = []
         logger.info("\tStarting cross validation")
+        warnings.filterwarnings("ignore", lineno=665)
         for i, (train, test) in enumerate(self.data_split.split(X, y)):
             logger.debug(f"\t\tTraining and scoring on fold {i}")
             X_train, X_test = X.iloc[train], X.iloc[test]
             y_train, y_test = y.iloc[train], y.iloc[test]
+            if self.problem_type in [ProblemTypes.BINARY, ProblemTypes.MULTICLASS]:
+                diff_train = set(y).difference(set(y_train))
+                diff_test = set(y).difference(set(y_test))
+                diff_string = f"Missing target values in the training set: {diff_train}. " if diff_train else ""
```

Comment:
Might be a good idea to specify the missing values are introduced after the data is split. As a user, I'd be confused about this message if I new my labels had values for each class.